docs/api-guide/viewsets.md

```diff
@@ -144,7 +144,7 @@ A more complete example of extra actions:
         queryset = User.objects.all()
         serializer_class = UserSerializer
 
-        @action(detail=True, methods=['post'])
+        @action(detail=True, methods=['post'], serializer_class=PasswordSerializer)
```

Comment:
Can we then use `self.get_serializer()` in the method body? (Otherwise the kwarg isn’t doing anything...)